Sources/BraintreePayPalMessaging/BTPayPalMessagingClient.swift

```diff
@@ -141,10 +142,12 @@ extension BTPayPalMessagingClient: PayPalMessageViewEventDelegate, PayPalMessage
     }
 
     public func onSuccess(_ paypalMessageView: PayPalMessages.PayPalMessageView) {
+        apiClient.sendAnalyticsEvent(BTPayPalMessagingAnalytics.succeeded)
```

Comment:
Typically we have used the `notify` pattern in other PRs. Open to using that pattern here if folks prefer but to me it seemed odd to extract out the delegate call below and the analytics event for just success and error while keeping the rest of the delegate calls above when we aren't sending analytics.